pvlib/atmosphere.py

```diff
@@ -174,41 +167,56 @@ def get_relative_airmass(zenith, model='kastenyoung1989'):
     other models use true (not refraction-adjusted) zenith angle. Apparent
     zenith angles should be calculated at sea level.
 
+    Comparison among several models is reported in [10]_.
+
     References
     ----------
-    .. [1] Fritz Kasten. "A New Table and Approximation Formula for the
-       Relative Optical Air Mass". Technical Report 136, Hanover, N.H.:
-       U.S. Army Material Command, CRREL.
+    .. [1] Fritz Kasten, "A New Table and Approximation Formula for the
+       Relative Optical Air Mass," CRREL (U.S. Army), Hanover, NH, USA,
+       Technical Report 136, 1965.
+       :doi:`11681/5671`
 
     .. [2] A. T. Young and W. M. Irvine, "Multicolor Photoelectric
        Photometry of the Brighter Planets," The Astronomical Journal, vol.
        72, pp. 945-950, 1967.
+       :doi:`10.2172/110366`
```

Comment:
hmm... this doi did not work for me. Try:
```suggestion
       :doi:`10.1086/110366`
```